Packs/CommonTypes/IndicatorFields/indicatorfield-Kill_Chain_Phases.json

```diff
@@ -0,0 +1,49 @@
+{
+    "associatedTypes": null,
+    "associatedToAll": true,
```

Comment:
is this intended?

Reply:
"associatedToAll": true? yes.

Reply:
I dont see in the issue that it was intended to be assisted to all.
Where was this decided?

Reply:
This is a field that has "STIX kill chain phases" version. when we talked about it with @ytauman, we said that new fields will be the same besides removing the "STIX" prefix. So because this is the type of "STIX kill chain phases" it's the same one as for "kill chain phases". but I can double-check with @ytauman.